src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -657,7 +657,12 @@ public CitizenData getCitizenData()
     {
         if (capability == CapabilityItemHandler.ITEM_HANDLER_CAPABILITY)
         {
-            return (T) new InvWrapper(getCitizenData().getInventory());
+            final CitizenData data = getCitizenData();
+            if (data == null) {
+              return null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 17. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)